src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js

```diff
@@ -634,7 +634,7 @@ const vakantieverhuur = {
       type: 'caution',
       value: [
         'Bel nu met 14 020',
-        'Vraag naar team Vakantieverhuur. U wordt direct doorverbonden met een medewerker. Handhaving gaat, indien mogelijk, meteen langs.',
+        'Vraag naar team Vakantieverhuur. U wordt direct doorverbonden met een medewerker. Handhaving gaat, indien mogelijk, binnen 24 uur langs.',
```

Comment:
Ik zou zeggen: "Handhaving *komt* [...] langs".

Reply:
I agree, but that's not up to me ;) Will forward it.